go/libkb/usercard_cache.go

```diff
@@ -0,0 +1,62 @@
+package libkb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/keybase/client/go/protocol/keybase1"
+
+	"stathat.com/c/ramcache"
+)
+
+// UserCardCache caches keybase1.UserCard objects in memory.
+type UserCardCache struct {
+	cache *ramcache.Ramcache
+}
+
+// NewUserCardCache creates a UserCardCache.  keybase1.UserCards will expire
+// after maxAge.
+func NewUserCardCache(maxAge time.Duration) *UserCardCache {
```

Comment:
what about switching users from alice to bob.  Will the card cache be blown away then? I think you need to, since the card has the "follows you" bits